src/lib/versionSelect.js

```diff
@@ -0,0 +1,38 @@
+const { prompt } = require('enquirer');
+const promptOpts = require('./prompts');
+const request = require('request-promise-native');
+const config = require('config');
+const APIError = require('./apiError');
+
+async function getSwaggerVersion(versionFlag, key, allowNewVersion) {
```

Comment:
The versions API isn't specific to Swagger/OAS so we should rename this

```suggestion
async function getProjectVersion(versionFlag, key, allowNewVersion) {
```